.github/jobs/webstandard.sh

```diff
@@ -92,62 +92,64 @@ if [ "$NUM_ERRORS" -ne 0 ]; then
 fi
 section_end
 
-if [ "$TEST" = "w3cval" ]; then
-    section_start "Remove files from upstream with problems"
-    rm -rf localhost/domjudge/doc
-    rm -rf localhost/domjudge/css/fontawesome-all.min.css*
-    rm -rf localhost/domjudge/bundles/nelmioapidoc*
-    rm -f localhost/domjudge/css/bootstrap.min.css*
-    rm -f localhost/domjudge/css/select2-bootstrap*.css*
-    rm -f localhost/domjudge/css/dataTables*.css*
-    rm -f localhost/domjudge/jury/config/check/phpinfo*
-    section_end
-
-    section_start "Install testsuite"
-    cd "$DIR"
-    wget https://github.com/validator/validator/releases/latest/download/vnu.linux.zip
-    unzip -q vnu.linux.zip
-    section_end
-
-    FLTR='--filterpattern .*autocomplete.*|.*style.*|.*role=tab.*|.*descendant.*|.*Stray.*|.*attribute.*|.*Forbidden.*|.*stream.*|.*obsolete.*'
-    for typ in html css svg
-    do
-        section_start "Analyse with $typ"
-	# shellcheck disable=SC2086
-        "$DIR"/vnu-runtime-image/bin/vnu --errors-only --exit-zero-always --skip-non-$typ --format json $FLTR "$URL" 2> result.json
-	# shellcheck disable=SC2086
-	NEWFOUNDERRORS=$("$DIR"/vnu-runtime-image/bin/vnu --errors-only --exit-zero-always --skip-non-$typ --format gnu $FLTR "$URL" 2>&1 | wc -l)
-        FOUNDERR=$((NEWFOUNDERRORS+FOUNDERR))
-        python3 -m "json.tool" < result.json > "$ARTIFACTS/w3c$typ$URL.json"
-        trace_off; python3 gitlab/jsontogitlab.py "$ARTIFACTS/w3c$typ$URL.json"; trace_on
+if [ "$TEST" != "none" ]; then
```

Comment:
Reviewing would be much easier if you exit here early.

So if TEST==none: exit(number_of_errors)